### PR TITLE
include model results in model config thing

### PR DESCRIPTION
**Pull request by benfoley**
This gets training results after training is done, and includes them in the GUI model table.

Plus a couple of other fixes.

training-dashboard

### CODE REVIEW

**Comment by benfoley** [comment]
When merged this will close #34

File: elpis/gui/src/index.css [hunk 1]
```diff
@@ -245,7 +245,9 @@ label.pad-right,
     color: #8abc1e;
 }
 
-.choose-dataset button { max-width: 300px }
+.choose-dataset button { 
+  max-width: 300px
```
Comment (nicklambourne) [inline elpis/gui/src/index.css:249]: indent should be 4 spaces, not 2

**Review by nicklambourne — APPROVED** [review]
One tiny nitpick, otherwise code looks good and have run/tested with no issues.

**Comment by benfoley** [comment]
Should be a lint for that ;-D

**Review by nicklambourne — APPROVED** [review]

**Pull request merged**